controllers/authController.js

```diff
@@ -2,64 +2,73 @@ const jwt = require('jsonwebtoken');
 const bcrypt = require('bcrypt');
 
 const UserModel = require('../models/userModel');
-// const UserValidationSchema = require('../validations/userValidation');
+const UserValidator = require('../validations/userValidation');
 
 const controller = {
   register: async (req, res) => {
-    // TO DO: validation
-    const validatedValues = req.body;
+    const validatedResults = UserValidator.register.validate(req.body);
+    if (validatedResults.error) {
+      return res.status(400).json({
+        error: 'Invalid input',
+      });
+    }
 
+    const validatedValues = validatedResults.value;
```

Comment:
Can you just use validatedResults or validatedValues? Both makes it confusing. Could start with let validatedResults = null then assign a new value to it if there is no error